.changes/next-release/feature-AWSSDKforJavav2-f7b0ca8.json

```diff
@@ -0,0 +1,5 @@
+{
+    "type": "feature",
```

Comment:
They wouldn't work correctly without each other, so I didn't feel like it was right to split them. I have another PR coming with an unrelated bug fix, since that one could be split.